### PR TITLE
Send to board

### PR DESCRIPTION
**Pull request by GrahamBriggs**
Changed openbci_serial_board .h/.cpp to return the response when you call send_to_board

### CODE REVIEW

**Review by Andrey1994 — CHANGES_REQUESTED** [review]
cyton test failed with check for welcome chars: https://github.com/brainflow-dev/brainflow/pull/222/checks?check_run_id=1851561593

File: src/board_controller/openbci/openbci_serial_board.cpp [hunk 1]
```diff
@@ -165,26 +194,12 @@ int OpenBCISerialBoard::prepare_session ()
...
-    tmp_array[tmp_id] = '\0';
-
-    if (strncmp ((const char *)tmp_array, "Failure", 7) == 0)
+    std::string response = read_serial_response ();
+    if (response.compare("Failure") == 0)
     {
         safe_logger (spdlog::level::err,
             "Board config error, probably dongle is inserted but Cyton is off.");
-        safe_logger (spdlog::level::trace, "read {}", tmp_array);
+        safe_logger (spdlog::level::trace, "read {}", response.c_str());
```
[the remark this answers is not in the document]
Comment (Andrey1994) [reply src/board_controller/openbci/openbci_serial_board.cpp:210]: https://brainflow.readthedocs.io/en/stable/BrainFlowDev.html#code-style

**Comment by GrahamBriggs** [comment]
Fixed up these items, will submit a new PR

**Comment by Andrey1994** [comment]
thanks! btw you can change existing PR, no need to create new one each time(just push to the branch and PR will be updated automatically)

**Pull request closed without merging**